BREAKING.md

```diff
@@ -20,8 +20,13 @@ It's important to communicate breaking changes to our stakeholders. To write a g
 ## 2.0.0-internal.3.0.0 Upcoming changes
 
 ## 2.0.0-internal.3.0.0 Breaking changes
+- [Make existing flag required](#make-existing-flag-required)
 - [Remove iframe-driver](#remove-iframe-driver)
 
+### Make existing flag required
+The `existing` flag has been updated to be expected and required in the `instantiateRuntime` method. The vast
```

Comment:
You need to specify the interface where this is happening. The public interface is what users care about, not the class implementation.